lib/elixir_internal_certificate_web/templates/user_search/_keyword_list.html.heex

```diff
@@ -0,0 +1,64 @@
+<table class="table table-hover" id="search-result-table">
+  <thead class="thead-default">
+    <tr>
+      <th>#</th>
+      <th>Keyword</th>
+      <th>Status</th>
+      <th>Details</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr role="button" class="align-middle">
+      <td scope="row">1</td>
+      <td>Mark</td>
+      <td class="text-success">Success</td>
+      <td>
+        <%=
+          button "Html response",
```

Comment:
small comment we can use the `link` https://hexdocs.pm/phoenix_html/Phoenix.HTML.Link.html#link/2 for the `get link`

Reply:
Good catch, updated in [1889475](https://github.com/nvminhtue/elixir_internal_certificate/pull/36/commits/1889475a2a989200399e58c18afa287a31772e39)